libpod/container_top.go

```diff
@@ -25,7 +25,7 @@ func (c *Container) GetContainerPids() ([]string, error) {
 // Gets the pids for a container without locking.  should only be called from a func where
 // locking has already been established.
```

Comment:
We'll have to be cognizant of this once things get rolling.  I understand the why, but am concerned about race conditions being added down the road.